src/inmanta/agent/config.py

```diff
@@ -116,9 +116,9 @@
     "config",
     "agent-repair-interval",
     600,
-    "The number of seconds between two repair runs (full deploy) of the agent. "
+    "The number of seconds between two repair runs (full deploy) of the agent or a cron expression of when to run the repair run"
```

Comment:
I think we need to be a bit more verbose on what this entails. The `auto_full_compile` environment setting already has some more details.